SampleApp/Info.plist

```diff
@@ -39,6 +39,7 @@
 		<string>UIInterfaceOrientationPortrait</string>
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
+		<string>UIInterfaceOrientationPortraitUpsideDown</string>
```

Comment:
Have you tested the SampleApp upside down?

Reply:
Yes I tested it. The UI is actually not rotating on iPhone it's only for iPad which must support all orientation.